src/autoconfig.cc

```diff
@@ -330,7 +330,7 @@ Autoconfig::onSend (fty_proto_t **message)
     log_debug("Decoded asset message - device name = '%s', type = '%s', subtype = '%s', operation = '%s'",
             device_name.c_str (), info.type.c_str (), info.subtype.c_str (), info.operation.c_str ());
     info.attributes = utils::zhash_to_map(fty_proto_ext (*message));
-    if (info.operation != "delete") {
+    if (info.operation != "delete" && streq (fty_proto_aux_string (*message, FTY_PROTO_ASSET_STATUS, "active"), "active")) {
```

Comment:
while you're editing the sources, could you please update "delete" to FTY_PROTO_ASSET_OP_DELETE?